chanfitness/chaneventstore.go

```diff
@@ -268,7 +259,19 @@ func (c *ChannelEventStore) consume(subscriptions *subscriptions) {
 			// A channel has been closed, we must remove the channel
 			// from the store and record a channel closed event.
 			case channelnotifier.ClosedChannelEvent:
-				c.closeChannel(event.CloseSummary.ChanPoint)
+				compressed := event.CloseSummary.RemotePub.SerializeCompressed()
```

Comment:
Question (not related to this line): since in `consume()` we read from many channels, could we run into event ordering issues?